gcloud/streaming/transfer.py

```diff
@@ -370,70 +379,57 @@ def GetRange(self, start, end=None, additional_headers=None,
         Args:
           start: (int) Where to start fetching bytes. (See above.)
           end: (int, optional) Where to stop fetching bytes. (See above.)
-          additional_headers: (bool, optional) Any additional headers to
-              pass with the request.
           use_chunks: (bool, default: True) If False, ignore self.chunksize
               and fetch this range in a single request.
 
         Returns:
           None. Streams bytes into self.stream.
         """
-        self.EnsureInitialized()
+        self._ensure_initialized()
         progress_end_normalized = False
         if self.total_size is not None:
-            progress, end_byte = self._NormalizeStartEnd(start, end)
+            progress, end_byte = self._normalize_start_end(start, end)
             progress_end_normalized = True
         else:
             progress = start
             end_byte = end
         while (not progress_end_normalized or end_byte is None or
                progress <= end_byte):
-            end_byte = self._ComputeEndByte(progress, end=end_byte,
+            end_byte = self._compute_end_byte(progress, end=end_byte,
                                              use_chunks=use_chunks)
-            response = self._GetChunk(progress, end_byte,
-                                       additional_headers=additional_headers)
+            response = self._get_chunk(progress, end_byte)
             if not progress_end_normalized:
-                self._SetTotal(response.info)
-                progress, end_byte = self._NormalizeStartEnd(start, end)
+                self._set_total(response.info)
+                progress, end_byte = self._normalize_start_end(start, end)
                 progress_end_normalized = True
-            response = self._ProcessResponse(response)
+            response = self._process_response(response)
             progress += response.length
             if response.length == 0:
-                raise exceptions.TransferRetryError(
+                raise TransferRetryError(
                     'Zero bytes unexpectedly returned in download response')
 
-    def StreamInChunks(self, additional_headers=None):
-        """Stream the entire download in chunks."""
-        # XXX:  this function should just go away during cleanup
-        self.StreamMedia(additional_headers=additional_headers,
-                         use_chunks=True)
-
-    def StreamMedia(self, additional_headers=None, use_chunks=True):
+    def stream_file(self, use_chunks=True):
         """Stream the entire download.
 
         Args:
-          additional_headers: (default: None) Additional headers to
-              include in fetching bytes.
           use_chunks: (bool, default: True) If False, ignore self.chunksize
               and stream this download in a single request.
 
         Returns:
             None. Streams bytes into self.stream.
         """
-        self.EnsureInitialized()
+        self._ensure_initialized()
         while True:
```

Comment:
"Unprotected" "infinite" loops make me uneasy. Fine for now, but in general prefer to avoid.

Feel much better after reflecting and realizing that the server on the other end won't go on forever.